### DISCUSSION TITLE
How to share tailwind config across multiple projects?

### DISCUSSION THREAD

**Discussion by ryanlloydfrench** [tailwindlabs/tailwindcss#2133] [Help]
Hi,

We are building a shared react component lib package which requires tailwind to be installed as a dependency. We also use a custom tailwind config. What is the best way to share a tailwind config across projects, ideally we want to keep them all in sync?

Thanks.

**Comment by gsimone** [comment]
You could host the config on npm as a module, or as a git submodule

**Comment by apgapg** [comment]
We can create presents for tailwind config

**Reply by C0ZEN** [reply]
Presets*
